lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -271,245 +283,169 @@ def test_unmatching_y_axis(self):
             self.plugin._spatial_domain_match()
 
 
-class Test__get_calibrated_forecast_predictors_mean(
+class Test__calculate_location_parameter_from_mean(
         SetupCoefficientsCubes, EnsembleCalibrationAssertions):
 
-    """Test the _get_calibrated_forecast_predictors_mean method."""
+    """Test the __calculate_location_parameter_from_mean method."""
 
     def setUp(self):
-        """Setup cubes and sundries for testing calibration of mean."""
+        """Set-up coefficients and plugin for testing."""
         super().setUp()
+
         self.optimised_coeffs = (
             dict(zip(self.coeffs_from_mean.coord("coefficient_name").points,
                      self.coeffs_from_mean.data)))
         self.plugin = Plugin()
         self.plugin.current_forecast = self.current_temperature_forecast_cube
 
-        self.expected_calibrated_predictor_mean = (
-            self.expected_calibrated_predictor_mean.flatten())
-
     @ManageWarnings(
         ignored_messages=["Collapsing a non-contiguous coordinate."])
     def test_basic(self):
-        """
-        Test that the expected values are returned by this function.
-        """
-        expected_forecast_predictors = (
-            self.current_temperature_forecast_cube.collapsed(
-                "realization", iris.analysis.MEAN))
-        predicted_mean, forecast_predictors = (
-            self.plugin._get_calibrated_forecast_predictors_mean(
+        """Test that the expected values for the location parameter are
+        calculated when using the ensemble mean. These expected values are
+        compared to the results when using the ensemble realizations to ensure
+        that the results are similar."""
+        location_parameter = (
+            self.plugin._calculate_location_parameter_from_mean(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _calculate_location_parameter_from_mean of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4714944)